keras/utils/data_utils.py

```diff
@@ -536,11 +536,12 @@ def stop(self, timeout=None):
 
 class GeneratorEnqueuer(SequenceEnqueuer):
     """Builds a queue out of a data generator.
+    The provided generator can be finite in which case the class will throw a StopIteration exception.
```

Comment:
Introduce a line break after the first docstring line, and format the line you added into 2 shorter lines. Add code delimiters ` around `StopIteration`.